src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/AnalyzeResult.cs

```diff
@@ -1,14 +1,15 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
 using System.Collections.Generic;
-using Microsoft.CodeAnalysis.Sarif;
-using Microsoft.DotNet.UpgradeAssistant.Dependencies;
 
 namespace Microsoft.DotNet.UpgradeAssistant.Analysis
 {
-    public interface IProcessResult
+    public record AnalyzeResult
     {
-        IList<Run> RunProcessResult(string toolName, Dictionary<string, IDependencyAnalysisState> analysisResults);
+        public string AnalysisFileLocation { get; init; } = string.Empty;
+
+        public IReadOnlyCollection<string> AnalysisResults { get; init; } = Array.Empty<string>();
```

Comment:
AnalysisResults in AnalyzeResult? These property names are a bit redundant.

Reply:
Naming is hard :P

Reply:
sure is! :)